src/app/[locale]/dashboard/_components/products/all-products-table.tsx

```
@@ -0,0 +1,83 @@
+import { deleteProduct } from "@lib/apis/dashboard/products/delete-product.api";
+import { Product } from "@lib/types/end-point-api/products";
+import { Pencil, Trash2 } from "lucide-react";
+import { useState, useTransition } from "react";
+import { toast } from "sonner";
+
+interface Props {
+  products: Product[];
+}
+
+export default function ProductsTable({ products }: Props) {
+    const [items, setItems] = useState(products);
```

ESLint

[error] 'items' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)